### PR TITLE
cp: implement -x option

### PR DESCRIPTION
**Pull request by grimreaper**
- implement the -x for cp, to avoid copying across mount points

### CODE REVIEW

File: bin/cp/cp.1 [hunk 1]
```diff
@@ -44,14 +44,14 @@
 .Op Fl fipv
 .Oo
 .Fl R
-.Op Fl H | L | P
+.Op Fl H | L | P | x
```
Comment (natano) [inline bin/cp/cp.1]: Is -x really mutually exclusive to -HLP? This seems wrong to me.

File: bin/cp/cp.1 [hunk 2]
```diff
@@ -44,14 +44,14 @@
...
 .Oc
 .Ar source target
 .Nm cp
 .Op Fl fipv
 .Oo
 .Fl R
-.Op Fl H | L | P
+.Op Fl H | L | P | x
```
Comment (natano) [inline bin/cp/cp.1]: Same thing here...

**Comment by natano** [comment]
The -x option also need to be show as extension to POSIX in the STANDARDS section of the manpage.

**Comment by natano** [comment]
usage() needs updating too.

**Comment by grimreaper** [comment]
updated

**Comment by Bluerise** [comment]
Committed to master by @natano, closing PR.

**Comment by natano** [comment]
Thank you for the diff! I have committed it to master (see f560499f638c80113b83ba9cbf98fc286f0f0c23).

Please note that I have changed `fts_options = FTS_XDEV;` to `fts_options |= FTS_XDEV;`, as the former is wrong, because it resets the other flags. You would have easily spotted the problem, if you tested the change - Please make sure to test your diffs before before putting them up for review :grinning:

**Pull request closed without merging**